lib/postgrex.ex

```diff
@@ -489,8 +518,10 @@ defmodule Postgrex do
         Enum.into(File.stream!("posts"), stream)
       end)
   """
-  @spec stream(DBConnection.t, iodata | Postgrex.Query.t, list, Keyword.t) ::
-    Postgrex.Stream.t
+  @spec stream(DBConnection.t(), iodata | Postgrex.Query.t(), list, [
+          execute_option | {:max_rows, pos_integer()}
+        ]) ::
+          Postgrex.Stream.t()
```

Comment:
this is abit ugly, not sure how to improve it. Introducing a `@type stream_option` just for this seems too much?

Reply:
Use when: `when option: execute_option | {:max_rows, pos_integer()}`.

Reply:
You can also force the formatter to split on each parameter instead of splitting inside the line. But using `when` looks better here.

Reply:
yup, already fixed in https://github.com/elixir-ecto/postgrex/pull/443/commits/43b52a384e9432c4071dd4950d42f859034363e5 :)